schemas/styles/csl-citation-label-format.rnc

```diff
@@ -0,0 +1,101 @@
+namespace a = "http://relaxng.org/ns/compatibility/annotations/1.0"
+namespace cs = "http://purl.org/net/xbiblio/csl"
+
+
+## cs:citation-label-format - Format for automatically generated citation-label
+div {
+  style.citation-label-format =
+    
+    ## Specify the format for automatically-generated citation-labels.
+    ## If unspecified, citation-labels are not automatically generated.
+    element cs:citation-label-format {
+      (
+       ## Specify a predefined label style
+       attribute label-style {
+         
+         ## Style used in citeproc-js for CLS 1.0.* styles
```

Comment:
It's more important on the docstrings to include an example so someone can easily understand what the format is.